pkg/auth/handler/session/get.go

```diff
@@ -47,15 +48,11 @@ const GetRequestSchema = `
 	"$id": "#SessionGetRequest",
 	"type": "object",
 	"properties": {
-		"session_id": { "type": "string" }
+		"session_id": { "type": "string", "minLength": 1 }
```

Comment:
In most places, we should also include `"required": ["session_id"]` to ensure, for example, `session_id` is specified and is not empty.